RxSwiftExt.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "RxSwiftExt"
-  s.version      = "6.0.1"
+  s.version      = "6.1.0"
```

Comment:
I think we'll have to make this 6.1.1 - 
The 6.1.0 release is already locked to a commit without the updated pod spec, so `pod lib lint` won't let me upload it to the Specs repo